tests/90jira/SYN-627.pl

```diff
@@ -23,7 +25,9 @@
             my $room_id = $_;
 
             matrix_send_room_text_message( $user, $room_id, body => "$room_id" );
-         } @rooms );
+         } ( head -1, @rooms) );
```

Comment:
Ohyes; head is not tail. Oops. :)
In that case, you could use the  `@rooms[0..@rooms-2]` notation,or just stick with `List::Slice` as it was.